centralsettlement/chart-handler-transfersettlement/values.yaml

```diff
@@ -0,0 +1,308 @@
+              messageAsJSON: true
+              sync: true
+              consumeTimeout: 1000
+            rdkafkaConf:
+              socket_keepalive_enable: true
+      settlementwindow:
+        close:
+          config:
+            options:
+              mode: 2
+              batchSize: 1
+              pollFrequency: 10
+              recursiveTimeout: 100
+              messageCharset: 'utf8'
+              messageAsJSON: true
+              sync: true
+              consumeTimeout: 1000
+            rdkafkaConf:
+              socket_keepalive_enable: true
+    producer:
+      notification:
+        event:
+          config:
+            options:
+              messageCharset: 'utf8'
+            rdkafkaConf:
+              event_cb: true
+              compression_codec: 'none'
+              retry_backoff_ms: 100
+              message_send_max_retries: 2
+              socket_keepalive_enable: true
+              batch_num_messages: 100
+              dr_cb: false
+              socket_blocking_max_ms: 1
+              queue_buffering_max_ms: 1
+              broker_version_fallback: '0.10.1.0'
+              api_version_request: true
+            topicConf:
+              request_required_acks: 'all'
+              partitioner: 'murmur2_random'
+      settlementwindow:
+        close:
+          config:
+            options:
+              messageCharset: 'utf8'
+            rdkafkaConf:
+              event_cb: true
+              dr_cb: true
+              socket_keepalive_enable: true
+            topicConf:
+              request_required_acks: "all"
+              partitioner: 'murmur2_random'
+  ## DB Configuration
+  # db_type can either be 'postgres' or 'mysql'. Ensure the correct DB is enabled and configured below: postgresql.enabled or mysql.enabled
+  db_type: 'mysql'
+  # db_driver can either be 'pg' or 'mysql'. Ensure the correct corresponding db_type above has been set.
+  db_driver: 'mysql'
+  db_host: '$release_name-centralledger-mysql'
+  db_port: 3306
+  db_user: central_ledger
+  db_password: oyMxgZChuu
+  db_database: central_ledger
+  db_connection_pool_min: 10
+  db_connection_pool_max: 30
+  db_acquire_timeout_millis: 30000
+  db_create_timeout_millis: 30000
+  db_destroy_timeout_millis: 5000
+  db_idle_timeout_millis: 30000
+  db_reap_interval_millis: 1000
+  db_create_retry_interval_millis: 200
+  db_debug: false
+  # Api Handler Configuration
+  handlers:
+    disabled: false
+    api:
+      disabled: false
+    settings:
+      scripts_folder: '/opt/central-settlement/scripts/transferSettlementTemp'
+  # Settlement Window aggregation Configuration
+  window_aggregation:
+    retry_count: 3
+    retry_interval: 3000
+  hub_participant:
+    id: 1
+    name: hub
+  log_level: info
+  log_transport: file
+
+  error_handling:
+    include_cause_extension: false
+    truncate_extensions: true
+
+  rules: 
+    interchangeFeeCalculation.js: |
+      const payerFspId = transfer.payer.partyIdInfo.fspId
```

Comment:
This is not a valid "rule" file as its missing all the header information.

Please update with the contents of the actual `interchangeFeeCalculation.js:` file contents.